tests/snapshots/execution_spec__test-scalar-email-invalid.md_assert_0.snap

```diff
@@ -0,0 +1,27 @@
+---
+source: tests/execution_spec.rs
+expression: response
+---
+{
+  "status": 200,
+  "headers": {
+    "content-type": "application/json"
+  },
+  "body": {
+    "data": null,
+    "errors": [
+      {
+        "message": "internal: invalid value for scalar \"Email\", expected \"FieldValue::Value\"",
```

Comment:
The first part of the error still makes sense. The latter -  `expected \"FieldValue::Value\"` doesn't seem to make much sense to me.

Reply:
That error is thrown by async_graphql itself.. I don't think we have control over it

Reply:
We need to have control over error messages.